meta-client/src/main/java/org/apache/helix/metaclient/constants/MetaClientConstants.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.helix.metaclient.constants;
+
+public interface MetaClientConstants {
```

Comment:
Add apache license, same for a few other classes

Reply:
TFTR. Updated